cmd/sealer/cmd/build.go

```diff
@@ -55,15 +58,9 @@ build without base:
 
 build with args:
 	sealer build -f Kubefile -t my-kubernetes:1.19.8 --build-arg MY_ARG=abc,PASSWORD=Sealer123 .
-
 `,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		var (
-			buildContext = "."
```

Comment:
could we add a default value for build context ? if so, user can write one less parameter.

Reply:
Good idea. While I think we could keep the stable and fixed input. In this place, I pay more emphasis on the fixed format  than less parameters.

Reply:
correct, i will update the related doc later.